Polkadot spec wiki/guide integration (#4009)
w3f/polkadot-wiki: docs/build/build-protocol-info.md

## Extrinsics and Events

### Block Format

A Polkadot block consists of a block header and a block body. The block body is made up of
extrinsics representing the generalization of the concept of transactions. Extrinsics can
contain any external data the underlying chain wishes to validate and track.

The block header is a 5-tuple containing the following elements:

- `parent_hash`: a 32-byte Blake2b hash of the SCALE encoded parent block header.
- `number`: an integer representing the index of the current block in the chain. It is equal to
  the number of the ancestor blocks. The genesis state has number 0.
- `state_root`: the root of the Merkle tree, used as storage for the system.
- `extrinsics_root`: field which is reserved for the Runtime to validate the integrity of the
  extrinsics composing the block body.
- `digest`: field used to store any chain-specific auxiliary data, which could help the light
  clients interact with the block without the need of accessing the full storage as well as
  consensus-related data including the block signature.

> When a node creates or receives a new block, it must be announced to the network. Other nodes within

Reviewer: ```suggestion
Each block created or received by a node must be gossiped to the newtwork (i.e. to the other nodes). Other nodes within
```
